arviz/data/io_pystan.py

```diff
@@ -451,15 +474,6 @@ def get_sample_stats(fit, log_likelihood=None):
         name = "diverging" if name == "divergent" else name
         data[name] = values
 
-    # log_likelihood
-    if log_likelihood is not None:
-        log_likelihood_data = get_draws(fit, variables=log_likelihood)
-        data["log_likelihood"] = log_likelihood_data[log_likelihood]
-
-    # lp__
-    stat_lp = get_draws(fit, variables="lp__")
-    data["lp"] = stat_lp["lp__"]

```

Comment:
Are we putting `lp` back to `posterior` or keeping here? It should not go to `log_likelihood`

Reply:
I moved it to sample stats